pulp_container/app/registry_api.py

```diff
@@ -760,6 +761,10 @@ def get(self, request, path, pk=None):
         try:
             blob = models.Blob.objects.get(digest=pk, pk__in=repository_version.content)
         except models.Blob.DoesNotExist:
+            if pk == EMPTY_BLOB:
+                return redirects.distribution.redirect_to_content_app(
+                    f"{settings.CONTENT_ORIGIN}/pulp/container/{path}/blobs/{pk}"
+                )
```

Comment:
Would this work too?
```suggestion
                return redirects.redirect_to_content_app("blobs", pk)
```

Reply:
yes it should, i was wrong copy paste >.>
thanks for pointing this out!